modules/gapi/perf/cpu/gapi_core_perf_tests_cpu.cpp

```diff
@@ -266,10 +266,13 @@ INSTANTIATE_TEST_CASE_P(LUTPerfTestCustomCPU, LUTPerfTest,
 
 
 INSTANTIATE_TEST_CASE_P(ConvertToPerfTestCPU, ConvertToPerfTest,
-    Combine(Values(CV_8UC3, CV_8UC1, CV_16UC1, CV_32FC1),
-        Values(CV_8U, CV_16U, CV_16S, CV_32F),
-        Values(szSmall128, szVGA, sz720p, sz1080p),
-        Values(cv::compile_args(CORE_CPU))));
+    Combine(Values(AbsExact().to_compare_f()),
+            Values(CV_8UC3, CV_8UC1, CV_16UC1, CV_16SC1, CV_32FC1),
+            Values(CV_8U, CV_16U, CV_16S, CV_32F),
+            Values(szSmall128, szVGA, sz720p, sz1080p),
+            Values(2.5, 1.0, -1.0),
+            Values(250.0, 0.0, -128.0),
```

Comment:
So we have `9x` more _performance_ tests now, right? Not sure if it is a good idea. How did the execution time change?

Reply:
Answered above